packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FlutterWebView.h

```diff
@@ -27,6 +27,13 @@ NS_ASSUME_NONNULL_BEGIN
 - (UIView*)view;
 
 - (void)onMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result;
+
+- (NSURLRequest*)buildNSURLRequest:(NSDictionary<NSString*, id>*)arguments;
+
+- (void)onLoadUrl:(FlutterMethodCall*)call result:(FlutterResult)result;
+
+- (void)onLoadRequest:(FlutterMethodCall*)call result:(FlutterResult)result;
```

Comment:
These methods (since they are part of the public header file) should include API documentation according to [Google's Objective-C style guidelines](https://google.github.io/styleguide/objcguide.html#declaration-comments).

Reply:
The headers were made public to make them available for testing. The headers have now been moved to a separate header file specifically for testing.